From fcafcbd005b8bf43101c08906857d30c7276f422 Mon Sep 17 00:00:00 2001
From: Jamie Strandboge <jamie@ubuntu.com>
Date: Thu, 11 May 2017 16:29:43 +0200
Subject: [PATCH 19/33] apparmor, virt-aa-helper: Allow access to
 /sys/bus/usb/devices

Required to generate profiles allowing usb passthrough.

Bug-Ubuntu: https://bugs.launchpad.net/bugs/565691

Forwarded: no (part of continuous upstreaming effort)
Signed-off-by: Stefan Bader <stefan.bader@canonical.com>
---
 examples/apparmor/usr.lib.libvirt.virt-aa-helper | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/examples/apparmor/usr.lib.libvirt.virt-aa-helper b/examples/apparmor/usr.lib.libvirt.virt-aa-helper
index 94e8e88..df4f9c6 100644
--- a/examples/apparmor/usr.lib.libvirt.virt-aa-helper
+++ b/examples/apparmor/usr.lib.libvirt.virt-aa-helper
@@ -23,6 +23,8 @@ profile virt-aa-helper /usr/{lib,lib64}/libvirt/virt-aa-helper {
   # for hostdev
   /sys/devices/ r,
   /sys/devices/** r,
+  /sys/bus/usb/devices/ r,
+  /sys/bus/usb/devices/** r,
   deny /dev/sd* r,
   deny /dev/vd* r,
   deny /dev/dm-* r,
-- 
2.7.4

